HealthMonitoring.AcceptanceTests/Scenarios/Selenium/Dashboard_page.cs

```diff
@@ -0,0 +1,77 @@
+﻿using LightBDD;
+
+namespace HealthMonitoring.AcceptanceTests.Scenarios.Selenium
+{
+    [FeatureDescription(
+        @"In order to view all minitoring services
+        As User 
+        I want to open dashboard page")]
+    public partial class Dashboard_page
+    {
+        [Scenario]
+        public void Homepage_link_test()
+        {
+            Runner.RunScenario(
+                _ => Given_dashboard_page(),
+                _ => When_user_clicked_on_home_link(),
+                _ => Then_home_page_is_opened()
```

Comment:
The structure of When and Then wording should be corrected.
When - some action that is going to happen now  - When_user_clicks_on_home_link(),
Then - what should happen as a result of it - Then_home_page_should_open()

See https://www.agilealliance.org/glossary/gwt/